resourcing.mdx

```diff
@@ -6,14 +6,27 @@ icon: "desktop"
 
 ## Running Locally
 
-When running locally through Docker, we recommend making at least 4vCPU cores and 10GB of RAM available to Docker 
-(16GB is preferred). This can be controlled in the **Resources** section of the Docker Desktop settings menu.
+When running locally through Docker, we recommend allocating at least 4 vCPU cores and 10GB of RAM to Docker (16GB is preferred). You can adjust these settings in the **Resources** section of the Docker Desktop settings menu.
+
+To prevent Danswer from consuming excessive resources and potentially impacting your system's performance, you can set resource limits on the Docker containers. This is done by applying the `docker-compose.resources.yml` file when starting Danswer. This file allows you to control and limit the resources used by each Danswer component.
+
+To apply resource limits:
+
+1. Use the `docker-compose.resources.yml` file provided in the `deployment/docker_compose` directory. This file contains default resource limits and reservations for each service.
+
+2. When starting Danswer, include the resource file in your Docker Compose command:
+
+   ```bash
+   docker-compose -f docker-compose.dev.yml -f docker-compose.resources.yml up
```

Comment:
can we make this match the cmds in the quickstart (e.g. `-p danswer-stack` + remove the `-` in `docker-compose`)?